docs/index.rst

```diff
@@ -9,6 +9,7 @@ Manticore is a symbolic execution tool for analysis of binaries and smart contra
    :maxdepth: 2
    :caption: Contents:
 
+   verifier
```

Comment:
This may need to be CLI-tools
And then inside that we should have manpage-like documentation on how to use the commandline tools for each tool. (So far we have plain manticore and manticore-verifier )